test/trento/infrastructure/commanded/event_handlers/activity_log_event_handler_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Trento.Infrastructure.Commanded.EventHandlers.ActivityLogEventHandlerTest do
+  @moduledoc false
+  use Trento.DataCase
+
+  import Trento.Factory
+
+  alias Trento.ActivityLog.ActivityLog
+  alias Trento.Infrastructure.Commanded.EventHandlers.ActivityLogEventHandler
+
+  test "should log a domain event" do
```

Comment:
Pretty useless test :see_no_evil:, as we don't have anything really interesting happening here, except from calling the ActivityLogger. I can get rid of this.